samples/notebooks/python/1-basic-loading-the-kernel.ipynb

```diff
@@ -168,7 +168,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.10"
+   "version": "3.11.3"
```

Comment:
Is there something specific in 3.11.3 that we're using?

Reply:
this is just an internal log generated by VSCode, not a requirement. You can ignore it